scripts/zones/Batallia_Downs/npcs/Geomantic_Reservoir.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+-- Area: Batallia Downs
+-- NPC: Geomantic Reservoir
+-- Unlocks: Geo-Barrier
+-- !pos -677.645 -32.000 157.981
```

Comment:
All of these pos comments also need the zone number at the end so they can be traveled to from anywhere